<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../appicon.qrc">
    <normaloff>:/resource/appicon/APPIcon.png</normaloff>:/resource/appicon/APPIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,8">
    <item>
     <widget class="NavigationBar" name="treeView">
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="welcomePage">
       <property name="styleSheet">
        <string notr="true">#welcomePage
{
	background-color: white;
	 border: 1px solid #007BFF; 
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Constantia</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Welcome to the My Chat Application Server！</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="userAccountPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="userInfoPage">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tableView_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavigationBar</class>
   <extends>QTreeView</extends>
   <header location="global">navigationbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../appicon.qrc"/>
 </resources>
 <connections/>
</ui>
